src/libs/highlevel/README.md

```diff
@@ -271,11 +272,12 @@ typedef enum { A, B, C } MyEnum;
 ElektraError * error = NULL;
 Elektra * elektra = elektraOpen ("/sw/org/myapp/#0/current", NULL, &error);
 
-// Read raw int value
-int value = elektraGetEnumInt (elektra, "message");
+// Read enum
 MyEnum enumValue = elektraGetEnum(elektra, "message", MyEnum);
 
-// enumValue == (MyEnum) value
+// Alternative: Read raw int value and then cast
+int value = elektraGetEnumInt (elektra, "message");
+enumValue == (MyEnum) value
```

Comment:
shouldn't that be a comment, so that the example is valid C code?

Reply:
Thank you, can you please do it? (Is now in master)